app/javascript/components/patient/Viewers.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Alert } from 'react-bootstrap';
+import { Beforeunload } from 'react-beforeunload';
+
+import axios from 'axios';
+
+class Viewers extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  onUnload = () => {
+    axios.post(window.BASE_PATH + `/patients/${this.props.patient_id}/on_unload`);
+  };
+
+  render() {
+    return (
+      <React.Fragment>
+        {this.props.viewers?.length === 1 && (
+          <Alert className="mx-2 mb-4 text-truncate" key="multiple-warning" id="multiple-warning" variant="info">
```

Comment:
I just played around with the styling and you can remove the `mb-4` here and then make the following changes:
- Change `className="mx-2 mt-1 mb-4"` to `className="ml-2 mb-3"` [here](https://github.com/SaraAlert/SaraAlert/blob/1.29.0/app/javascript/components/patient/export/DownloadMonitoree.js#L63)
- Change `className="mx-1 mt-1 mb-4"` to `className="mx-2 mb-3"` [here](https://github.com/SaraAlert/SaraAlert/blob/1.29.0/app/javascript/components/patient/export/DownloadMonitoree.js#L71)

This fixes some screen resizing issues with those buttons too

Reply:
I like it! Changed in 0eb95ac2b0a272f0a45c7c8963cc370fb6cc48fb